package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "purescript-node-postgres",
+  "version": "5.0.0",
+  "description": "PureScript bindings for the pg PostgreSQL library",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "postinstall": "PATH=node_modules/.bin:$PATH bower install",
```

Comment:
npm automatically adds `node_modules/.bin` to the PATH.

Reply:
Ah, thanks @anttih. Hmm, I think I must have something misconfigured because that didn't work for me. In particular, `pulp` couldn't find `purs`. Still, I've addressed your comment in #27, does that work for you?